src/client/components/pages/entity-revisions.js

```diff
@@ -54,6 +69,14 @@ class EntityRevisions extends React.Component {
 		);
 	}
 
+/**
+* Renders the data related to Revision such as 'author' and 'date'.
+* It also displays the first revison note which is a summary of the changes
+* made in the revision.
+* @param {object} revision for the revison to be represented  by the
```

Comment:
I'd also remove the "for" here, and have the description start with "the revision..."